# Evaluating the Novel Clustering Method ScGMM-VGAE for Single-Cell RNA Sequencing Data
## Abstract
**Background:** The fundamental step in single-cell RNA-sequencing (scRNA-seq) computational analysis is cell clustering, which groups sets of cells into meaningful subpopulations without using truth labels. A Gaussian mixture model-based variational graph autoencoder (scGMM-VGAE) was developed to cluster the scRNA data in our previous study. However, its performance evaluation and compatibility with imputation have not been fully studied.  
**Objectives:** In this study, we aim to make a comprehensive evaluation of scGMM-VGAE with other state-of-the-art clustering methods. The evaluations focus on four aspects: clustering performance for labeled real and simulated data, imputationâ€™s effect on clustering methods, clustering performance for unlabeled data, and method stability and time efficiency. 
**Methods:** In addition to scGMM-VGAE, the other four baseline clustering methods are scVI, Cell-VGAE, Seurat, and DCA + Leiden. The datasets include three real labeled data, two real unlabeled data, and three simulated datasets with different cell numbers. The data imputation is conducted by scImpute, and the simulated datasets are generated using SPARSim. The clustering performances are evaluated by adjusted Rand index, normalized mutual information, and silhouette score metrics. 
**Results:** Based on the three metrics, ScGMM-VGAE outperforms the other four baseline methods on all the labeled, unlabeled, and simulated datasets. Meanwhile, scGMM-VGAE is less sensitive to imputation compared to other methods and shows great stability and time efficiency. 
**Conclusion:** ScGMM-VGAE successfully combines an unsupervised deep learning model (VGAE) with a statistical model (GMM) to enhance the clustering performance of scRNA data. The algorithm can be applied to future scRNA-seq research. 
![image](https://user-images.githubusercontent.com/105017237/185806633-800d3680-2775-4f60-b3be-b78e6a6f3ef3.png)
